src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -452,6 +474,18 @@ const PopularRepositoriesTable = () => {
               </FlexItem>
             </InputGroupItem>
           </InputGroup>
+          {searchQuery !== '' && (
+            <ChipGroup categoryName='Name/URL'>
+              <Chip key='search_chip' onClick={() => setSearchQuery('')}>
+                {searchQuery}
+              </Chip>
+            </ChipGroup>
+          )}
+          {(debouncedSearchValue !== '' && searchQuery !== '') || (
+            <Button className={classes.clearFilters} onClick={clearFilters} variant='link' isInline>
+              Clear filters
+            </Button>
+          )}
```

Comment:
This code always shows, you need to hide the clear filters button when no search query is made.

Reply:
![Uploading Screenshot 2024-01-22 at 8.28.26 AM.png…]()